src/pkg/cli/client/byoc/do/byoc.go

```diff
@@ -461,10 +451,11 @@ func (b *ByocDo) Follow(ctx context.Context, req *defangv1.TailRequest) (client.
 }
 
 func (b *ByocDo) TearDown(ctx context.Context) error {
-	_, err := b.BootstrapCommand(ctx, "down")
-	if err != nil {
-		return err
-	}
+	// FIXME: Do we need this? if so can we get it from remote?
+	// _, err := b.BootstrapCommand(ctx, "down")
+	// if err != nil {
+	// 	return err
+	// }
```

Comment:
Actually, is this behaviour consistent with the AWS teardown? aws provider only bring down the cd stack without removing the services afaik.